src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -133,8 +139,34 @@ $left-sidebar-z-index: 40 !default;
     }
 }
 
-@include large-size {
-    .bdl-CollapsibleSidebar-logo {
-        margin-left: $collapsible-side-bar-h-padding + $bdl-grid-unit * 2 - 1px /* border */;
+@include breakpoint($medium-screen) {
+    .Body.is-responsive-web {
+        .bdl-CollapsibleSidebar-wrapper {
+            width: 0;
+        }
+
+        .bdl-CollapsibleSidebar-footer {
+            display: none;
+        }
+
+        .bdl-CollapsibleSidebar-nav ul {
+            display: none;
+            // using fixed width due to performance issues during expand transition
+            width: calc(100vw - (#{$collapsible-side-bar-h-padding} * 2));
```

Comment:
I know you didn't come up with them, but these variable names could be better. `Sidebar` should be `sidebar` instead of `side-bar` and something like `padding-x` and `padding-y` seems much more clear than `h-padding` and `v-padding`. I actually didn't know what `h-padding` was without the context of `v-padding`---maybe because it is 2:50AM, or maybe because it's not a great name.

Reply:
Agreed. Also looks like one of the variable names has a typo: $colla**s**psible-side-bar-logo-v-margin. We can address this in a future PR.